src/OrbitGl/Track.cpp

```diff
@@ -148,12 +148,16 @@ void Track::Draw(Batcher& batcher, TextRenderer& text_renderer, uint64_t current
     Vec2 content_top_right(top_left[0] + size_[0] - right_margin,
                            top_left[1] - label_height + top_margin);
 
-    DrawTriangleFan(batcher, rounded_corner, top_left, background_color, -90.f, track_z);
-    DrawTriangleFan(batcher, rounded_corner, tab_top_right, background_color, 180.f, track_z);
-    DrawTriangleFan(batcher, rounded_corner, tab_bottom_right, background_color, 0, track_z);
-    DrawTriangleFan(batcher, rounded_corner, content_bottom_left, background_color, 0, track_z);
-    DrawTriangleFan(batcher, rounded_corner, tab_bottom_right, background_color, 0, track_z);
-    DrawTriangleFan(batcher, rounded_corner, content_bottom_left, background_color, 0, track_z);
+    DrawTriangleFan(batcher, rounded_corner, top_left, GlCanvas::kBackgroundColor, -90.f, track_z);
```

Comment:
Why is this GlCanvas::BackgroundColor and not `track_background_color`?

Reply:
Because we have to paint the top left corner with the background color from canvas on top of the track color to make the grey rounding corner

Reply:
I am still a bit confused why this is changed now. But I trust that you checked the outcome.

Reply:
I've messed up. I wrote track background color thinking that was the same than GlCanvas::kBackgroundColor, it's a revert + a renaming.